client/src/tokens/colorVariables.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
This whole file is very clever :clap: It did take me a while to understand it, though :)

Would you mind adding a few more comments? Maybe some sample inputs and outputs?

Reply:
Good call! I’ve added much more, and changed the code a bit so it’s hopefully less clever.